<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTableWidgetClass</class>
 <widget class="QWidget" name="AddTableWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_tablename">
     <property name="text">
      <string>TableName:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_tablename"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_primkeyType">
     <property name="text">
      <string>PrimaryKeyType:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_keyTypes"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_primkey">
     <property name="text">
      <string>PrimaryKey:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_primkey"/>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_certain">
     <property name="text">
      <string>certain</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>52</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>cencel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
